src/components/Selector/NativeSelect.tsx

```diff
@@ -0,0 +1,63 @@
+import { Listbox } from "@headlessui/react";
+import { DrawerIcon } from "components/Icon";
+import { AriaAttributes, forwardRef } from "react";
+import { OptionType, ValKey } from "types/components";
+import FocusableInput from "./FocusableInput";
+import { styles } from "./constants";
+
+type Props<V extends ValKey> = {
+  "aria-invalid"?: AriaAttributes["aria-invalid"];
+  disabled?: boolean;
+  id?: string;
+  options?: OptionType<V>[];
+  value?: OptionType<V>;
+  onChange: (value: OptionType<V>) => void;
+};
+
+export const NativeSelect = forwardRef(function Select<V extends ValKey>(
+  props: Props<V>,
+  ref: React.Ref<HTMLInputElement>
+) {
+  return (
+    <Listbox
```

Comment:
technically, this is still controlled component (manifested by `value + onChange` ), which result to this bug

REPRO: 
- go to `admin/banking/new`
- select PHP, amount:50, 
- select E-wallet

<img width="1202" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/1393204a-9c1a-4d05-89c2-ffae594bc6ae">

<img width="463" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/496cd198-2b56-469b-959c-dfa72b2a2562">


headlessui/listbox can however be [uncontrolled](https://headlessui.com/react/listbox#using-as-an-uncontrolled-component) but I'm not sure if it could be used just like native `<select/>`
```
<select {...register(...)} /> 
```

Reply:
@ap-justin by "uncontrolled" I meant "not dependent on ('controlled by') `react-hook-form`"

About the bug - good catch, will address.

Reply:
> headlessui/listbox can however be [uncontrolled](https://headlessui.com/react/listbox#using-as-an-uncontrolled-component) but I'm not sure if it could be used just like native `<select/>`
> ```
> <select {...register(...)} /> 
> ```

I did try some things out unsuccessfully, but I just thought of an another approach to attempt, will test it out :+1:

Reply:
No luck.
Created a new branch to show the changes (see [native-select-with-register](https://github.com/AngelProtocolFinance/angelprotocol-web-app/tree/native-select-with-register)).

The attempt consisted of updating [NativeSelect to be able to accept `{...register(...)}` props](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/native-select-with-register/src/components/Selector/NativeSelect.tsx).
Then [I updated the relevant usage of the component so that no `Controller` is necessary](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/native-select-with-register/src/components/BankDetails/RecipientDetails/RecipientDetailsForm/RecipientDetailsForm.tsx#L131-L143).
But unfortunately, `react-hook-form > register` always expects to work with native components (its `onChange` always expects native event parameter, see [docs](https://react-hook-form.com/docs/useform/register)), so it didn't know how to handle the actual `ValKey` value.

Resolved the bug you reported by setting the initial selector value to be an empty string.